import-scripts/fetch-dmp-data-for-import.sh

```diff
@@ -195,43 +195,43 @@ printTimeStampedDataProcessingStepMessage "MSKIMPACT data processing"
 printTimeStampedDataProcessingStepMessage "Darwin CAISIS fetch for mskimpact"
 $JAVA_BINARY $JAVA_DARWIN_FETCHER_ARGS -s mskimpact -d $MSK_IMPACT_DATA_HOME -c
 if [ $? -gt 0 ] ; then
-    cd $MSK_IMPACT_DATA_HOME ; $HG_BINARY update -C ; find . -name "*.orig" -delete
+    cd $DMP_DATA_HOME ; $GIT_BINARY reset HEAD --hard
     sendPreImportFailureMessageMskPipelineLogsSlack "MSKIMPACT Darwin CAISIS Fetch"
 else
     FETCH_DARWIN_CAISIS_FAIL=0
     echo "committing darwin caisis data"
-    cd $MSK_IMPACT_DATA_HOME ; $HG_BINARY commit -m "Latest MSKIMPACT Dataset: Darwin CAISIS"
+    cd $MSK_IMPACT_DATA_HOME ; $GIT_BINARY add ./* ; $GIT_BINARY commit -m "Latest MSKIMPACT Dataset: Darwin CAISIS"
```

Comment:
this might add files we don't want to check in - e.g we export redcap files above this (line ~180)... when we do a git add ./* it will add the redcap exported files.

Reply:
@averyniceday @sheridancbio This follows the same behavior we were doing before actually. In mercurial we are committing the redcap raw clinical/timeline files and then removing them in a later commit. I think it's fine to leave it as is.